src/loader-components/clinical-lowlight-theme/PropsTableModule.scss

```diff
@@ -0,0 +1,6 @@
+:local{
```

Comment:
Whoops, looks like a few of these files were misnamed. `module` should be an extension

`PropsTableModule.scss` -> `PropsTable.module.scss`